frontend/src/components/app/app.tsx

```diff
@@ -1,9 +1,21 @@
-import { FC } from 'react';
+import { FC, useEffect } from 'react';
+import { useAppDispatch } from 'hooks/hooks';
+import { auth as authActions } from 'store/actions';
+
 import { Routes, Route } from 'components/common/common';
 import { AppRoute } from 'common/enums/enums';
 import { Auth } from 'components/auth/auth';
 
 const App: FC = () => {
+  const dispatch = useAppDispatch();
+  const userId = Number(localStorage.getItem('userId')); // need change
```

Comment:
so change it?

you can't do any side effect directly in the components. you can do it only in the actions